.circleci/continue_config.yml

```diff
@@ -354,20 +354,6 @@ jobs:
           name: Pip-compile requirements file
           command: conda activate kedro_builder; make pip-compile
 
-  build_docs:
```

Comment:
About your first question: I think it has to be configured at the repo level, from what I remember it can be done.

About linkcheck, using the new `build.commands` we could move it there yes. That way we could keep "all things docs" in one place.

Since this needs an update anyway, I'll have a look at enabling that in this PR, but if something unexpected happens I'll defer it.

Reply:
Cool. Do you think docs build failing is something that _should_ block a PR from merging? If so then we should probably get @merelcht or someone else with the right privileges to try to enable that.

Reply:
As a general case, I'm on the fence - but if we don't have that, _at least_ we should be extra careful before releases.

Docs issues gave us a bad week last one, but now that we are fully in control, I'd be fine to have it as a mandatory check, and just folks pinging me if they don't understand how to fix the docs. I defer the final decision to @merelcht

Reply:
On second thought, yeah please let's have this as a blocker 🙏🏽